pkg/resmap/resmap.go

```diff
@@ -119,7 +120,7 @@ func NewResourceSliceFromPatches(
 
 		res, err := newResourceSliceFromBytes(content)
 		if err != nil {
-			return nil, err
+			return nil, internal.ErrorHandler(err, path)
```

Comment:
There might be other places we parse YAML files and catch the error. Can you also apply this error handler for them?

Reply:
Sure